services/centralized-kubecost/0.20.2/defaults/cm.yaml

```
@@ -0,0 +1,143 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: centralized-kubecost-0.20.2-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    ---
+    cost-analyzer:
+      fullnameOverride: "kommander-kubecost-cost-analyzer"
+      global:
+        prometheus:
+          fqdn: http://kubecost-prometheus-server.kommander.svc.cluster.local
+          enabled: false
+
+        thanos:
+          enabled: true
+          queryService: http://kommander-kubecost-thanos-query-http.kubecost.svc.cluster.local:10902
+          # The wait time before Kommander begins querying cost data for all attached clusters
+          queryOffset: 5m
+          query:
+            deploymentAnnotations:
+              secret.reloader.stakater.com/reload: kommander-kubecost-thanos-client-tls
+
+        grafana:
+          enabled: false
+          # Use kommander monitoring Grafana instance
+          domainName: centralized-grafana.${releaseNamespace}.svc.cluster.local
+
+      # For Thanos Installs, Allow Higher Concurrency from Cost-Model
+      # Still may require tweaking for some installs, but the thanos-query-frontend
+      # will greatly assist in reduction memory bloat in query.
+      kubecostModel:
+        maxQueryConcurrency: 5
+        # This configuration is applied to thanos only. Expresses the resolution to
+        # use for longer query ranges. Options: raw, 5m, 1h - Default: raw
+        maxSourceResolution: 5m
+
+      ingress:
+        enabled: true
+        # We can remove this once https://github.com/kubecost/cost-analyzer-helm-chart/pull/1132 is released.
+        className: ""
+        annotations:
+          kubernetes.io/ingress.class: kommander-traefik
+          ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
+          traefik.ingress.kubernetes.io/router.tls: "true"
+          traefik.ingress.kubernetes.io/router.middlewares: "${releaseNamespace}-stripprefixes@kubernetescrd,${releaseNamespace}-forwardauth@kubernetescrd"
+        paths:
+          - "/dkp/kommander/kubecost/frontend/"
+        hosts:
+          - ""
+        tls: []
+
+      kubecostDeployment:
+        labels:
+          vendor.kubecost.io/partner: d2iq
+
+      podSecurityPolicy:
+        enabled: false
+
+      grafana:
+        # These values are set so that kubecost grafana dashboards are installed.
+        # Grafana itself is not installed.
+        sidecar:
+          image: kiwigrid/k8s-sidecar:1.3.0
```

We need this to be bumped to `1.12.3` to mitigate the CVEs.
```suggestion
          image: kiwigrid/k8s-sidecar:1.12.3
```